app/services/websocket_server.py

```diff
@@ -22,6 +23,56 @@
 def connect(sid, environ):
     print("connect ", sid)
 
+# get returnTicker, grabs data from tickers table
+async def get_tickers(token_hexstr):
+
+    # init
+    where = ''
+    placeholder_args = []
+
+    # if token is passed in, add where addr = token
+    if token_hexstr:
+        where = '("token_address" = $1)'
+        placeholder_args = [Web3.toBytes(hexstr=token_hexstr), ]
+
+    # connect to db and grab data from tickers table
+    async with App().db.acquire_connection() as conn:
+        return await conn.fetch(
+            """
+            SELECT *
+            FROM tickers
+            WHERE {}
+            ORDER BY token_address DESC
+            """.format(where),
+            *placeholder_args)
+
+# format ticker information to camelcase from underscore, sanitize, add modified date
+"""
+    sample return data:
+    ETH_0x8f3470a7388c05ee4e7af3d01d8c722b0ff52374:
+   { tokenAddr: '0x8f3470a7388c05ee4e7af3d01d8c722b0ff52374',
+     quoteVolume: 1000.1,
+     baseVolume: 212.3,
+     last: 0.245,
+     percentChange: 0.0047,
+     bid: 0.243,
+     ask: 0.246,
+     modified: 2018-01-15T15:53:00},
+"""
+def format_ticker(ticker):
+    contract = ERC20Token(ticker["token_address"])
+    return { "{}_{}".format("ETH", Web3.toHex(ticker["token_address"]) : {
+        "tokenAddr": Web3.toHex(ticker["token_address"]),
+        "quoteVolume": str(contract.denormalize_value(ticker["quote_volume"])),
+        "baseVolume": str(contract.denormalize_value(ticker["base_volume"])),
+        "last": str(contract.denormalize_value(ticker["last"])),
+        "percentChange": str(contract.denormalize_value(ticker["percent_change"])),
+        "bid": str(contract.denormalize_value(ticker["bid"])),
+        "ask": str(contract.denormalize_value(ticker["ask"])),
+        "modified": ticker["modified"]
+        # "modified": datetime.now().isoformat()
```

Comment:
Dead lines should be removed.

Reply:
Placeholder for when I write the code that updates the data. Will remove at that time.